src/saveload/saveload.h

```diff
@@ -323,7 +323,8 @@ enum SaveLoadVersion : uint16 {
 
 	SLV_GS_INDUSTRY_CONTROL,                ///< 287  PR#7912 and PR#8115 GS industry control.
 	SLV_VEH_MOTION_COUNTER,                 ///< 288  PR#8591 Desync safe motion counter
-	SLV_INDUSTRY_TEXT,                      ///< 289  PR#8576 v1.11 Additional GS text for industries.
+	SLV_INDUSTRY_TEXT,                      ///< 289  PR#8576 Additional GS text for industries.
```

Comment:
worth noting that it's also 1.11 beta2/RC1?

Reply:
We never mark betas in this list, and I guess we could leave behind it was used for RC1. Not very important if you ask me, but sure.